devbox.json

```diff
@@ -2,11 +2,11 @@
   "packages": [
     "addlicense@1.0.0",
     "bats@1.3.0",
-    "github:nixos/nixpkgs/d7f28652048b3bed6a512542e62ea1a50691e349#buf",
-    "clang@11.1.0",
+    "github:nixos/nixpkgs/2920b6fc16a9ed5d51429e94238b28306ceda79e#buf",
+    "llvmPackages_14.clangUseLLVM@14.0.6",
     "gci@0.9.1",
-    "github:nixos/nixpkgs/deb3d80ae0ccafe4f19d3edf32e2eb7fda283978#go",
-    "github:nixos/nixpkgs/22540da6c19a41512dd6bc575858e1215baeb3f9#golangci-lint",
+    "go@1.20.5",
```

Comment:
I think Alan recently bumped go to 1.20.6 everywhere

Reply:
And also buf to 1.24 - https://github.com/gravitational/teleport/pull/29081

Reply:
Yep, both go and buf are still lagging here. go in particular usually has a ~2ish week lag time for version bumps.